test/runtests.jl

```diff
@@ -11,7 +11,9 @@ using MCMCChains
 using NamedArrays
 using Optim
 using Optimization
-using OptimizationOptimJL
+using OptimizationBBO: BBO_separable_nes
+using OptimizationOptimJL: LBFGS, NelderMead, Fminbox, IPNewton
+using OptimizationNLopt: NLopt.LD_TNEWTON_PRECOND_RESTART
```

Comment:
No, it's actually used in the tests for Optimisation.jl, [here](https://github.com/TuringLang/Turing.jl/blob/472f6babd3c9c3ac1b77523627797df7aabab20f/test/optimisation/Optimisation.jl#L344). I wanted to check that things work for algorithms from a couple of different packages.

Relatedly, I might have a think about if we could restructure the tests into submodules, so that imports like these could only be in scope for the tests that actually need them, and be closer to where they are used.

Reply:
Yup, doing that now as part of the move to qualified references.